corehq/apps/reports/filters/case_list.py

```diff
@@ -19,10 +19,11 @@ def static_options(self):
         options = super(CaseListFilterUtils, self).static_options
         # replace [Active Mobile Workers] and [Deactivated Mobile Workers] with case-list-specific options
         assert options[0][0] == "t__0"
+        # If static_options include GROUPS_AND_LOCATIONS, the case list should not show it
         return [
             ("all_data", _("[All Data]")),
             ('project_data', _("[Project Data]"))
-        ] + options[2:]
+        ] + options[2:6]
```

Comment:
At this point I'd prefer explicitly defining all static options here, even if it means that they are duplicated.  As-is, it drops the first two options, adds two more, and drops any options beyond the 6th index.  This is fairly obscure, and prone to failure if someone changes the options in the super class.  They could be put in global constants if that's preferable.

Reply:
That makes sense, I'm going to do this in a separate PR

Reply:
Addressed and cleaned up code here: https://github.com/dimagi/commcare-hq/pull/22586